mat64/matrix.go

```diff
@@ -781,6 +767,87 @@ func MaybeFloat(fn func() float64) (f float64, err error) {
 	return fn(), nil
 }
 
+// Norm returns the specified (induced) norm of the matrix a. See
+// https://en.wikipedia.org/wiki/Matrix_norm for the definition of an induced norm.
+//
+// Valid norms are:
+//    1 - The maximum absolute column sum
+//    2 - Frobenius norm, the square root of the sum of the squares of the elements.
+//  Inf - The maximum row sum.
+// Norm will panic with ErrNormOrder if an illegal norm order is specified.
+func Norm(a Matrix, norm float64) float64 {
+	var n lapack.MatrixNorm
```

Comment:
Factor this block L779-L802 into a prep function that is called immediately prior to each lapack64 call. At the moment, there's a lot of work being done that might not be used.